src/pyramid_retry/__init__.py

```diff
@@ -213,7 +227,7 @@ def is_last_attempt(request):
     if attempt is None or attempts is None:
         return True
 
-    return attempt + 1 == attempts
+    return attempt + 1 == attempts  # type: ignore[no-any-return] # TODO: what's the best way to determine these?
```

Comment:
I'm not certain how to inspect these, since both `attempt` and `attempts` _could_ be `t.Any`, due to the untyped nature of `request.environ` (a dict) and they are being set earlier on.

I contemplated adding `assert isinstance(attempt, int); assert isinstance(attempts, int)` but that felt clunky too.